keras_cv/layers/preprocessing/random_color_jitter.py

```diff
@@ -66,51 +61,59 @@ class RandomColorJitter(VectorizedBaseImageAugmentationLayer):
             is used, a value between `0.0` and the passed float is sampled.
             In order to ensure the value is always the same, please pass a tuple
             with two identical floats: `(0.5, 0.5)`.
+        value_range: Optional list/tuple of 2 floats for the lower and upper limit
```

Comment:
any reason to re-order this?  consistency I suppose?  Either way, this parameter needs to be required, not optional

Reply:
Yes, for consistency.
`value_range` is a bit orderless across preprocessing layers

I have a list of this:

## At 1st argument:
- AugMix
- Equalization
- Posterization
- RandAugment
- RandomChannelShift
- Solarization
- RandomColorJitter

## Not at 1st argument
- RandomBrightness
- RandomContrast
- RandomHue
- RandomSharpness

Should `value_range` be at 1st argument or next to `seed`/`**kwargs`?

I have no idea. I just make RandomColorJitter consistent with its sub-layers.

Reply:
I'm going to make all `value_range` next to `seed` or `**kwargs` and remove default values.

This should improve user experience when using preprocessing layers.

Reply:
Done.
Also I change the position of `value_range` and remove default value.

Now only RandomSharpness has `value_range` after `factor`.